ldm/dream/args.py

```diff
@@ -174,31 +174,37 @@ def dream_prompt_str(self,**kwargs):
         switches.append(f'-W {a["width"]}')
```

Comment:
This is OK for now, but we need to have a common library for storing, reading and writing the args. Otherwise when any new feature is added, we will have to change four different files. I think that @Kyle0654 has unified the args, and I'm happy to redo the arg processor (or use @Kyle0654 's) if it isn't fit for what all sections of the code need to do.